numba/tests/test_tuples.py

```diff
@@ -745,8 +745,10 @@ def foo():
 
         with self.assertRaises(errors.UnsupportedError) as raises:
             foo()
-
-        msg = "op_LIST_EXTEND at the start of a block"
+        if utils.PYVERSION == (3, 9):
+            msg = "op_LIST_EXTEND at the start of a block"
+        elif utils.PYVERSION == (3, 10):
+            msg = "SPECIALSTRING"
```

Comment:
We should change this to something more informative.

Reply:
done in 8c7a69d804b9213e2a46c28eccede4eb930d437b

Reply:
Is 8c7a69d on this branch?

Reply:
now as a5497934cc3faa8d46e12b97d59f166faf5e4161